docs/articles/documentation/guides/advanced-guides/speed-up-test-execution.md

```diff
@@ -103,3 +104,37 @@ This approach ensures that the page model object is created only once per test r
 > Important! If you export the page model class and create an instance in every test file, JavaScript creates a new page object per test file. This increases test execution time and memory consumption.
 
 For more on page models, read [Page Model](../../guides/concepts/page-model.md).
+
+## Enable Server-Side Caching
+
+> Important! Server-side caching is **experimental**.
+>
+> Disable caching if you run into compatibility issues with your tests.
```

Comment:
I propose to rephrase this sentence since the feature is disabled by default.

`Don't use caching if you run into compatibility issues...`